scripts/new_migration_clean.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+MIGRATION_DIRECTORY='app/migrations'
+LAST_MIGRATION_FILE_NAME=$(ls -t $MIGRATION_DIRECTORY | head -n1)
+FILE_PATH=${MIGRATION_DIRECTORY}/${LAST_MIGRATION_FILE_NAME}
+
+if grep -q ' : void' $FILE_PATH; then
```

Comment:
i would run the target `phpcsfix` before this script to avoid this CS step :)
wdyt?